streams/src/test/java/org/apache/kafka/streams/kstream/internals/suppress/KTableSuppressProcessorMetricsTest.java

```diff
@@ -136,16 +135,15 @@
     public void shouldRecordMetrics() {
         final String storeName = "test-store";
 
-        final StateStore buffer = new InMemoryTimeOrderedKeyValueBuffer.Builder(storeName)
+        final StateStore buffer = new InMemoryTimeOrderedKeyValueBuffer.Builder<>(storeName, String(),
```

Comment:
nit: move `Serde.String()` into next line (also, I would prefer to not have static import `Serde` but prefix. I was initially confuse why this compiles and why it's not `new String()`... (because the method name starts with capital `S`...)

Reply:
Oh! That's awful. I didn't realize the ambiguity until you pointed it out. I'll add the `Serdes` qualifier back.